### PR TITLE
Build detail

### PR DESCRIPTION
**Pull request by carletex**
Iteration v1 of a single Build page. We can iterate with Andrea later on, for now I just did a basic restyling. 

Update: Andrea sent me a new body gradient color for dark mode, so I changed it and reverted the alternativeBaseColor thing. Now I think it looks better.

image
image
image

### CODE REVIEW

File: packages/react-app/components/BuildCard.jsx [hunk 1]
```diff
@@ -139,7 +139,7 @@ const BuildCard = ({ build, userProvider, userRole, onUpdate }) => {
         <Spacer />
         <ButtonGroup mt={3}>
           <NextLink href={`/build/${build.id}`} passHref>
-            <Button variant="outline" size="sm" isFullWidth colorScheme="customBaseColorScheme">
+            <Button as="a" variant="outline" size="sm" isFullWidth colorScheme="customBaseColorScheme">
```
Comment (carletex) [inline packages/react-app/components/BuildCard.jsx:142]: This allows you to do CTLR + click on the "button".

**Comment by technophile-04** [comment]
Lol just got notification you pushed, everything looking great !! 

Just pushing a nitpick of centering things on BuildDetailsHeader card in mobile

**Comment by carletex** [comment]
> Just pushed a nitpick of centering things on BuildDetailsHeader card in mobile at https://github.com/scaffold-eth/buidlguidl-v3/commit/08411cfea3b040b8f44452c157571b6195623ad9

Looks cool, thank you!

Just pushed a change to display the build cards better on mobile. https://github.com/scaffold-eth/buidlguidl-v3/pull/100/commits/58cc998b9907235b5865098ce6600bf3e72d39bc

**Review by technophile-04 — APPROVED** [review]
> Just pushed a change to display the build cards better on mobile. https://github.com/scaffold-eth/buidlguidl-v3/commit/58cc998b9907235b5865098ce6600bf3e72d39bc

Nice ! also the update to dark gardient play's will with other stuff too now ! 

Merging this, Thanks !!

**Pull request merged**